pkg/keyspace/tso_keyspace_group_test.go

```
@@ -18,9 +18,11 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/pingcap/kvproto/pkg/keyspacepb"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)